qiskit_sphinx_theme/footer.html

```diff
@@ -37,30 +24,19 @@
       {%- endif %}
     {%- endif %}
 
-    {%- if build_id and build_url %}
-      {% trans build_url=build_url, build_id=build_id %}
-        <span class="build">
-          Build
-          <a href="{{ build_url }}">{{ build_id }}</a>.
-        </span>
-      {% endtrans %}
-    {%- elif commit %}
-      {% trans commit=commit %}
-        <span class="commit">
-          Revision <code>{{ commit }}</code>.
-        </span>
-      {% endtrans %}
-    {%- elif last_updated %}
```

Comment:
This looks like a cleanup. Do you know what the purpose of these lines is?

Reply:
i think it was just copy pasted over from the pytorch theme, I couldn't see anything actually implemented with this for our packages